e2e/e2etests/test_message_passing_zevm.go

```diff
@@ -0,0 +1,332 @@
+package e2etests
+
+import (
+	"fmt"
+	"math/big"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/zeta-chain/zetacore/e2e/contracts/testdapp"
+	"github.com/zeta-chain/zetacore/e2e/runner"
+	"github.com/zeta-chain/zetacore/e2e/utils"
+	cctxtypes "github.com/zeta-chain/zetacore/x/crosschain/types"
```

Comment:
```suggestion
	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
```
Just for consistency with the rest of the code